src/main/java/frc/robot/subsystems/arm/SubstationIntakeState.java

```diff
@@ -27,12 +27,9 @@ public void build() {
     
     @Override
     public void init() {
-
-        double modifier = RobotMap.arm.getSubstationDirectionModifier();
```

Comment:
If we have the ability to score out the back, we should be able to maintain the ablility to intake from the back at the substation as well. Make this return 1.0 for now.